doc/system-administration.md

```diff
@@ -0,0 +1,100 @@
+# System administration and troubleshooting
+
+The goal of this document is to provide enough information to allow a system administrator to maintain a production DRIVER system, and to aid in troubleshooting if something goes wrong. This assumes a production system has already been deployed using the default configuration.
+
+
+## General system architecture
+
+The default DRIVER production setup consists of three servers, all running Ubuntu 14.04 LTS:
+
+1. Database host
+   * `PostgreSQL` database for storing data
```

Comment:
I could see it being useful for people that are unfamiliar with these tools/frameworks have their first mention be a link to their documentation or homepage. This could prevent some confusion in cases where someone needs to search for documentation and has to parse through google results.